recipe/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set name = "snowflake" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}-packages
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d3fa09f1e939de69251d7400b2343893b249953ec90564031097e726c50e2c3c
+  patches:
+    - patches/0001-fix-pyproject.patch
+
+build:
+  number: 0
+  # snowflake.core not available for s390x
+  skip: true  # [py<38 or py>311 or s390x]
+
+requirements:
+  build:
+    - patch     # [not win]
+    - m2-patch  # [win]
+  host:
+    # this is needed here in the multi-output recipe
+    # to make the PYTHON variable being set for later
+    # in the script section
+    - python
+
+outputs:
+  - name: {{ name }}
+    build:
+      # snowflake.core not available for s390x
+      skip: true  # [py<38 or py>311 or s390x]
+      script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vv
+    requirements:
+      host:
+        - python
+        - hatchling
+        - pip
+      run:
+        - python
+        - snowflake.core =={{ version }}
```

Comment:
I wonder if snowflake.core should be an output of this recipe instead of the separate [feedstock](https://github.com/AnacondaRecipes/snowflake.core-feedstock/blob/main/recipe/meta.yaml)? But it makes sense if the versions are the same always

Reply:
No. `snowflake` is a namespace package. It should stay like it is now.